### DISCUSSION TITLE
new release?

### DISCUSSION THREAD

**Discussion by tessus** [orhun/rustypaste#375] [General]
While not a lot of features have been added since the last release, a lot of dependencies have been updated and the shuttle code has been removed.

Can we cut a new release?

**Comment by orhun** [comment]
Yup, makes sense. Do we want #335 in the release? It seems almost ready.

**Reply by tessus** [reply]
I guess that depends on the author of that PR, but yeah, if he finishes it in the next few days. On the other side he was not very reactive to your ping 2 days ago...
It's up to you.

**Reply by orhun** [reply]
Hm, let's wait until tomorrow. Do you want to maybe submit another PR with his changes + other fixes for moving that forward?

**Reply by tessus** [reply]
Unfortunately I don't have time the next few days, otherwise I would.

**Comment by orhun** [comment]
https://github.com/orhun/rustypaste/releases/tag/v0.16.0
